node/src/Pos/Wallet/Web/ClientTypes/Types.hs

```diff
@@ -304,6 +306,12 @@ data CTExMeta = CTExMeta
     , cexId          :: CId Addr
     } deriving (Show, Generic)
 
+data NewPaymentBatchInit = NewPaymentBatchInit
+    { npbFrom   :: CAccountId
+    , npbTo     :: [(CId Addr, CCoin)] -- TODO: use NonEmpty here
```

Comment:
have to remove TODO, its ok this way

Reply:
Why `NewPaymentBatchInit` and now simply `NewBatchPayment`?

Reply:
added to TODO

Reply:
Fixed